test/unit/writer_spec.js

```diff
@@ -194,6 +194,7 @@ describe("Writer", function () {
         "\n" +
         "789 0 obj\n" +
         "<< /XFA [(preamble) 123 0 R (datasets) 101112 0 R (postamble) 456 0 R]>>\n" +
+        "endobj\n" +
```

Comment:
Were we previously, in some cases, potentially generating invalid PDF documents when saving XFA forms or was this just an artifact of the dummy-data used in the unit-test?